docs/rulesets.md

```diff
@@ -158,12 +165,5 @@ Reading ruleset
 /rules/rule-with-invalid-enum/type 	 should be equal to one of the allowed values 
 ```
 
-**Note for developers**
```

Comment:
I removed this because I didn't understand what it was talking about. I think we only offer YAML rulesets right now anyhow, and we removed our TS to YAML scrupt, but what is a TS to JSON Schema script?  @chris-miaskowski do you know about this?